ReSharper.FSharp/src/FSharp/FSharp.Psi.Intentions/src/QuickFixes/AddMissingSeqFix.fs

```diff
@@ -0,0 +1,39 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings.FcsErrors
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Resources.Shell
+open JetBrains.ReSharper.Psi.ExtensionsAPI.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Util
+
+type AddMissingSeqFix(expr: IComputationExpr) =
+    inherit FSharpQuickFixBase()
+    
+    new(error: ConstructDeprecatedSequenceExpressionsInvalidFormError) =
+        AddMissingSeqFix(error.ComputationExpr)
+        
+    new(error: InvalidRecordSequenceOrComputationExpressionError) =
+        AddMissingSeqFix(error.ComputationExpr)
+
+    override x.Text = "Add missing 'seq'"
+
+    override x.IsAvailable _ =
+        isValid expr
+
+    override x.ExecutePsiTransaction _ =
+        use writeCookie = WriteLockCookie.Create(expr.IsPhysical())
+        use disableFormatter = new DisableCodeFormatter()
```

Comment:
Please try to do this without disabling the formatter. If there're issues, let's look at it together.